src/library/dlgtagfetcher.cpp

```diff
@@ -535,8 +537,16 @@ void DlgTagFetcher::slotNetworkResult(
         const QString& app,
         const QString& message,
         int code) {
-    QString cantConnect = tr("Can't connect to %1: %2");
-    loadingProgressBar->setFormat(cantConnect.arg(app, message));
+    QString cantConnect = tr("Can't connect to %1: %2").arg(app, message);
+    QFontMetrics metrics(loadingProgressBar->font());
+    QString elidedCantConnect = metrics.elidedText(
+            cantConnect,
+            Qt::ElideRight,
+            loadingProgressBar->width() - 4);
+    loadingProgressBar->setFormat(elidedCantConnect);
+    if (cantConnect.size() != elidedCantConnect.size()) {
```

Comment:
Is there the possibility of the edge case that the elided text has the same length as the non-elided text? Could this happen when the text gets elided but then the added ellipses pads it back out to the same length again? That would cause information to be missing (not much sure, but still a character or two) and no tooltip being shown. 
Unless we know this is a performance problem, I would just always set the tooltip. makes the code simpler and avoids the edge case.

Reply:
Right, that is likely, but no gurantee, because the elipsis can be either the … (https://www.compart.com/de/unicode/U+2026) or ... (three dots) depending on the font.